### ISSUE TITLE
[BUG]: Author with ID 0 does not exist

### ISSUE THREAD

**Issue by b101marc** [Readarr/Readarr#1270]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When attempting to manually import a book across a variety of file types, I am unable to select any files and instead am met with this error: `Author with ID 0 does not exist`
<img width="1161" alt="Screen Shot 2021-10-04 at 4 32 23 PM" src="https://user-images.githubusercontent.com/41177825/135938124-15836584-1259-4173-a457-3efdea05e51b.png">

### Expected Behavior

I am able to select the file and import the book manually as usual.

### Environment

```markdown
- OS: Ubuntu 18.04
- Readarr: 0.1.0.522
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser: Safari/Chrome/Firefox
```

### What branch are you running?

Master

### Anything else?

Seems totally random.

[AB#1707](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/1707)

**Comment by RetroDadson** [comment]
Trace Logs needed for this please

**Comment by bakerboy448** [comment]
Also update first; .522 is very old and very out of date  .980 is the current nightly, if you're going to run pre-alpha software (like .522 was/is) it's highly advisable to stay current and up to date.  .980 is more beta quality

PS there is no `master` branch of readarr nor `develop` for that matter

**Comment by b101marc** [comment]
What would be the best docker image to use? Currently I have `hotio/readarr:pr-native-mvc`. I will update and test again. Should the error persist I will provide trace logs.

**Comment by bakerboy448** [comment]
`nightly` -  PR branches should only be used for explicit testing when instructed by the dev team and promptly switched off of once devs indicate testing is complete  - which may be by that PR being merged into develop (nightly)

**Comment by b101marc** [comment]
I was able to update to nightly and the problem is no longer persisting. Thanks for your assistance.
